dist/iconpicker-1.5.0.js

```diff
@@ -1,439 +1,432 @@
+
+      // icons wrap
+      var iconWrap = '<div class="ip-icons-content" style="border-radius:' + borderRadius + ';">' +
+        '<div class="ip-icons-search"><input id="IconPickerSearch" type="text" spellcheck="false" autocomplete="off" placeholder="' + searchPlaceholder + '" style="border-radius:' + borderRadius + ';" /><i class="placeholder-icon fas fa-search"></i></div>' +
+        '<div class="ip-icons-search-results"></div>' +
+        '<div class="ip-icons-area">' +
+        loadingIndicator +
+        icons +
+        '<a class="ip-show-all-icons" style="border-radius:' + borderRadius + ';">' + buttonShowAll + '</a>' +
+        '</div>' +
+        '<div class="ip-icons-footer"><a class="cancel" style="border-radius:' + borderRadius + ';">' + buttonCancel + '</a></div>' +
+        '</div>';
+
+      // create the modal element
+      var IconPickerModal = document.createElement('div')
+      IconPickerModal.id = 'IconPickerModal';
+      IconPickerModal.innerHTML = iconWrap;
+
+      // body
+      var docBody = document.body;
+
+      // append to body
+      docBody.appendChild(IconPickerModal);
+
+      // get the modal element
+      var ipElement = document.getElementById(IconPickerModal.id);
+
+      // modal element display css
+      ipElement.style.display = 'block';
+
+      // heights
+      var ipHeight = parseInt(ipElement.offsetHeight);
+      var winHeight = parseInt(window.innerHeight);
+
+      // modal element position top css on
+      var liveScrollTop = parseInt(window.pageYOffset || document.documentElement.scrollTop);
+      var totalTopPos = liveScrollTop + ((winHeight - ipHeight) / 2);
+      if (winHeight + 20 <= ipHeight) {
+        totalTopPos = liveScrollTop;
+      }
+      ipElement.style.top = totalTopPos + 'px';
+      // modal element position top css off
+
+      // add css animation class to modal
+      ipElement.classList.add('animate');
+
+      // remove loading indicator on
+      var loadingElement = document.getElementById('IconPickerLoading');
+      var ltAnimate = setTimeout(function() {
+        loadingElement.classList.add('hide');
+        clearTimeout(ltAnimate);
+      }, 600);
+      var ltRemove = setTimeout(function() {
+        loadingElement.parentNode.removeChild(loadingElement);
+        clearTimeout(ltRemove);
+      }, 900);
+      // remove loading indicator off
+
+      // show all button click listener on
+      var showAllButtonElm = document.getElementById(IconPickerModal.id).getElementsByClassName('ip-show-all-icons')[0];
+      showAllButtonElm.addEventListener('click', function() {
+        ipElement.classList.add('show-all');
+        this.parentNode.removeChild(this);
+      }, false);
+      // show all button click listener off
+
+      // close and remove all on
+      var removeIpElement = function(delay) {
```

Comment:
And this is actually bug. Generally picker works so like yeah it's fine but there was error in console log and i didn't liked it. The pioint is that Your observer (while needed for svg conversion) **stacks** click events so when You click the fontawesome icon the event here is being called alot of times and js tries to remove non existing element in parent and thus crashes:

````
            var removeIpElement = function(delay) {
                ipElement.classList.remove('animate');
                setTimeout(function() {
                    let isElementInDom = ( null !== ipElement.parentNode );
                    //prevent handling removal if element is already gof out of daoml
                    if( isElementInDom ){
                        docBody.removeChild(ipElement);
                    }

                }, delay);
            };
````